Richmond_Library_App/views/login.py

```diff
@@ -25,4 +24,20 @@ def post(self, request):
 class Logout(View):
     def get(self, request):
         logout(request)
-        return render(request, "login.html", {"errorMessage": "You have been logged out!"})
\ No newline at end of file
+        return render(request, "login.html", {"errorMessage": "You have been logged out!"})
+    
+    # This is for relogging into the app after clicking the logout button
+    def post(self, request):
+        # grabs the username and password provided
+        # from the form on login.html
+        name = request.POST.get('username')
+        password = request.POST.get('password')
+        # This is now using Django's Authentification system.
+        request.session['name'] = name
+        request.session['password'] = password
+        user = authenticate(request, username=name, password=password)
+        if user is not None:
+            login(request, user)
+            return redirect('/home/')
+        else:
+            return render(request, "login.html", {"errorMessage": "Invalid username or password"})
```

Comment:
I noticed that when you try to login after logging out, there is an error. This fixes that error. So now we can login directly after logging out of the app. This only applies when not using a social account to login